Test/comp/DowncastClone.dfy

```diff
@@ -3,8 +3,9 @@
 // RUN: %dafny /noVerify /compile:4 /compileTarget:js "%s" >> "%t"
 // RUN: %dafny /noVerify /compile:4 /compileTarget:go "%s" >> "%t"
 // RUN: %dafny_0 /noVerify /compile:4 /compileTarget:java "%s" >> "%t"
+// RUN: %dafny /noVerify /compile:4 /compileTarget:py "%s" >> "%t"
```

Comment:
Can we switch this one and TailRecursion.dfy to `%testdafny` instead?

Reply:
Neither works because of a bug in `%testdafny`.

Reply:
Thanks for trying anyway! I'll address the issue with the `Feature` code as part of completing #1770.